### ISSUE TITLE
Color Contrast Issue – Page Meta Text

### ISSUE THREAD

**Issue by X-07-xqf** [X-07-xqf/2850final-project#72]
The .page-meta text (date display) does not meet WCAG 2.1 AA minimum contrast requirements.
